src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -62,7 +62,11 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
 
         // Ensure that the lombok config is set to emit @Generated annotations
         lombokPlugin.configureForJacoco()
-        lombokPlugin.generateLombokConfig.get().generateLombokConfig()
+        // If config generation is enabled, automatically produce an appropriate config file.
+        def generateLombokConfig = lombokPlugin.generateLombokConfig.get()
+        if (generateLombokConfig.isEnabled()) {
```

Comment:
What are the consequences for the plugin if this test evaluates to false?
That is, I presume that the Lombok configuration does something useful.  Is that functionality lost, or is it achieved in some other way?

Reply:
The object construction checker, if run, will not issue warnings on incorrect Lombok builders. As far as I know, that's the only checker that relies on this behavior.